manage-data/lifecycle/index-lifecycle-management/start-stop-index-lifecycle-management.md

```diff
@@ -1,72 +0,0 @@
----
```

Comment:
would it make sense to turn this into a snippet and have it in both locations instead?

Reply:
It would! Done. :-)